nebula_bench/scenarios/insert.py

```diff
@@ -20,3 +20,21 @@ class InsertScenario(BaseScenario):
     abstract = False
     csv_path = "social_network/dynamic/person.csv"
     rank = 9999
+
+class BatchInsertEdgeScenario(BaseScenario):
+    is_insert_scenario = True
+    nGQL = 'INSERT EDGE LIKES (creationDate) VALUES '
+    value = '{0}->{1}:(datetime("{2}"))'
+    abstract = False
+    csv_path = "social_network/dynamic/person_likes_comment.csv"
+    rank = 9999
+
+class InsertEdgeScenario(BaseScenario):
+    is_insert_scenario = False
+    nGQL = (
+        'INSERT EDGE LIKES (creationDate) VALUES {0}->{1}:(datetime("{2}"))'
+    )
+    abstract = False
+    csv_path = "social_network/dynamic/person_likes_comment.csv"
+    rank = 9999
+    
```

Comment:
please add extra line.
https://community.smartthings.com/t/why-does-smarthings-ide-show-an-extra-line-at-the-end-from-github/98034

Reply:
added